WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift

```diff
@@ -416,6 +416,14 @@ extension GutenbergViewController: GutenbergBridgeDelegate {
 
 extension GutenbergViewController: GutenbergBridgeDataSource {
 
+    func gutenbergLocale() -> String? {
+        return Locale.current.languageCode
```

Comment:
Hey @pinarol, @koke I think that we have a couple of special cases when we want to return languageCode + region or script code.

As you can see from this docs : https://developer.apple.com/library/archive/documentation/MacOSX/Conceptual/BPInternational/LanguageandLocaleIDs/LanguageandLocaleIDs.html

**REGION CASE**

`let locale = Locale.current.languageCode`
`let regionCode = Locale.current.regionCode`
`locale = locale + '-' + regionCode?.lowercased()` so that we can get e.g. `en-au`

'en-au', // English (Australia)
'en-ca', // English (Canada)
'en-gb', // English (UK)
'pt-br', // Portuguese (Brazil)

**SCRIPT CODE CASE**

`let locale = Locale.current.languageCode`
`let scriptCode = Locale.current.scriptCode`

zh-tw-> Chinese Traditional | zh-Hant
zh-cn -> Chinese Simplified | zh-Hans

So we need to map the `Hant` and `Hans` to `tw`or  `cn`

```
 e.g. if (scriptCode == 'Hant') {
      scriptCode = 'tw';
 }
```
`locale = locale + '-' + scriptCode`

Reply:
thanks for the heads up @marecar3  so if this is the expected format then we have to support that otherwise it will not work. @Tug can you confirm this is necessary also?

Reply:
Yeah, that's correct. I will do the same on Android side.

Reply:
updated this by using WordPressComLanguageDatabase().deviceLanguage.slug , looks like it is returning what we want https://github.com/Automattic/calypso-pre-oss/blob/72c2029b0805a73b749a2b64dd1d8655cae528d0/config/production.json#L86-L227

Reply:
@pinarol it looks like that link isn't working for me :(

Reply:
it basically returns language keys suitable with https://github.com/wordpress-mobile/gutenberg-mobile/pull/536/files#diff-db716afdbf5160927b4465dba3e62535R8

Reply:
Nice work @pinarol !!!